### PR TITLE
Change the link to the tutorial page

### PR DESCRIPTION
**Pull request by bson1012**
### Change Description ###

Updated the link to the tutorial page for  'Getting started with Raspberry PI 3&4'.
(from http://tizenschool.org/tutorial/137 to http://tizenschool.org/tutorial/191)

For instance,
 - Fix broken links of native application docs.
 -  ...

### CODE REVIEW

File: docs/application/get-started/iot.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ The following guide demonstrates how to create and run a basic Tizen .NET applic
 First, ensure that you have installed the latest version of Visual Studio Tools for Tizen. [**Learn more >**](../vstools/install.md)
 
 ## Hardware Setup for IoT Development
-To start developing Tizen IoT applications, follow the [guide](http://tizenschool.org/tutorial/137/) that demonstrates how to set up the Raspberry Pi board.
+To start developing Tizen IoT applications, follow the [guide](http://tizenschool.org/tutorial/191/) that demonstrates how to set up the Raspberry Pi board.
```
Comment (mijin85cho) [inline docs/application/get-started/iot.md:16]: Suggestion. How about changing the sentence like below?

To start developing Tizen IoT applications, see the [Get started guide](http://tizenschool.org/tutorial/191/) that demonstrates how to set up the Raspberry Pi 3 and Raspberry Pi 4.

Comment (bson1012) [reply docs/application/get-started/iot.md:16]: How about this sentence? :)


To start developing Tizen IoT applications, follow the [Get Started guide](http://tizenschool.org/tutorial/191/) that demonstrates how to set up the Raspberry Pi 3 or 4 board.

Comment (mijin85cho) [reply docs/application/get-started/iot.md:16]: @bson1012 Thanks for the modification. I think your is better. : ) Would you please apply your suggestion?

Comment (bson1012) [reply docs/application/get-started/iot.md]: @mijin85cho I updated the sentence. thank you! 👍

**Pull request merged**